src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -1033,7 +1033,12 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
     <value>This package is signed but not by a trusted signer.</value>
   </data>
   <data name="Warning_HttpServerUsage" xml:space="preserve">
-    <value>You are running the '{0}' operation with an 'http' source, '{1}'. Support for 'http' sources will be removed in a future version.</value>
+    <value>You are running the '{0}' operation with an 'HTTP' source, '{1}'. Non-HTTPS access will be removed in a future version. Consider migrating to an 'HTTPS' source.</value>
```

Comment:
I'm fine with it but was this phrasing run by PM first?

Reply:
This was a suggestion by Jon.
All of the warnings have this text right now.

Reply:
https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Commands/Strings.resx#L1035-L1042, https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Protocol/Strings.resx#L481-L483 already match this.

Reply:
Already as in 3 hours ago? :) Thanks for sharing!

Reply:
Commandline xplat has them too :D https://github.com/NuGet/NuGet.Client/commit/9e244ebc11baf68ea2da0b7595b11726d0da99e5

That was merged 3 days ago. 
I knew it was 3-4 files, but I couldn't remember which ones exactly so I linked the ones I could remember :D